From 5970f45fd498d29c58c258d47fdd5e0f44595e43 Mon Sep 17 00:00:00 2001
From: tangyongjie <80395553+yongjieTang@users.noreply.github.com>
Date: Tue, 13 Jan 2026 15:55:56 +0800
Subject: [PATCH 5/5] Change COREPACK_ENABLE_NETWORK to 1 form 0

---
 tools/buildtools_helper.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/buildtools_helper.py b/tools/buildtools_helper.py
index e63fc95a..9731ec65 100644
--- a/tools/buildtools_helper.py
+++ b/tools/buildtools_helper.py
@@ -7,7 +7,7 @@ import os
 import sys
 
 def directory_really_is_buildtools(directory):
-  return os.path.exists(os.path.join(directory, 'gn')) and os.path.exists(os.path.join(directory, 'ninja'))
+  return os.path.exists(os.path.join(directory, 'corepack')) and os.path.exists(os.path.join(directory, 'node'))
 
 def get_buildtools_path():
   """Search for buildtools """
-- 
2.50.1 (Apple Git-155)

