From d4a01fa81297b517d046a4741a6515782b2f7ab1 Mon Sep 17 00:00:00 2001
From: zsy-jason <186247581+zsy-jason@users.noreply.github.com>
Date: Mon, 16 Mar 2026 23:00:39 +0800
Subject: [PATCH] adapt modp with lynx prefix

---
 base64.cc | 20 ++++++++++----------
 1 file changed, 10 insertions(+), 10 deletions(-)

diff --git a/base64.cc b/base64.cc
index 380e528bc..e0b079e3e 100644
--- a/base64.cc
+++ b/base64.cc
@@ -35,9 +35,9 @@ std::string Base64Encode(span<const uint8_t> input) {
 }
 
 void Base64EncodeAppend(span<const uint8_t> input, std::string* output) {
-  // Ensure `modp_b64_encode_data_len` will not overflow.
+  // Ensure `lynx_modp_b64_encode_data_len` will not overflow.
   CHECK_LE(input.size(), MODP_B64_MAX_INPUT_LEN);
-  size_t encode_data_len = modp_b64_encode_data_len(input.size());
+  size_t encode_data_len = lynx_modp_b64_encode_data_len(input.size());
 
   const size_t after_size =
       base::CheckAdd(encode_data_len, output->size()).ValueOrDie();
@@ -46,7 +46,7 @@ void Base64EncodeAppend(span<const uint8_t> input, std::string* output) {
   span<const char> read = base::as_chars(input);
   span<char> write = base::span(*output).last(encode_data_len);
 
-  const size_t written_size = modp_b64_encode_data(
+  const size_t written_size = lynx_modp_b64_encode_data(
       write.data(),  // This must point to `encode_data_len` many chars.
       read.data(), read.size());
   // If this failed it would indicate we wrote OOB or left bytes uninitialized.
@@ -63,16 +63,16 @@ bool Base64Decode(std::string_view input,
                   std::string* output,
                   Base64DecodePolicy policy) {
   std::string decode_buf;
-  decode_buf.resize(modp_b64_decode_len(input.size()));
+  decode_buf.resize(lynx_modp_b64_decode_len(input.size()));
 
   // Does not NUL-terminate result since result is binary data!
-  size_t written_size = modp_b64_decode(decode_buf.data(), input.data(),
+  size_t written_size = lynx_modp_b64_decode(decode_buf.data(), input.data(),
                                         input.size(), GetModpPolicy(policy));
 
   // Forgiving mode requires whitespace to be stripped prior to decoding.
   // We don't do that in the above code to ensure that the "happy path" of
   // input without whitespace is as fast as possible. Since whitespace in input
-  // will always cause `modp_b64_decode` to fail, just handle whitespace
+  // will always cause `lynx_modp_b64_decode` to fail, just handle whitespace
   // stripping on failure. This is not much slower than just scanning for
   // whitespace first, even for input with whitespace.
   if (written_size == MODP_B64_ERROR &&
@@ -87,10 +87,10 @@ bool Base64Decode(std::string_view input,
                 &input_without_whitespace);
     // This means that the required size to decode is at most what was needed
     // above, which means `decode_buf` will fit the decoded bytes at its current
-    // size and we don't need to call `modp_b64_decode_len()` again.
+    // size and we don't need to call `lynx_modp_b64_decode_len()` again.
     CHECK_LE(input_without_whitespace.size(), input.size());
     written_size =
-        modp_b64_decode(decode_buf.data(), input_without_whitespace.data(),
+        lynx_modp_b64_decode(decode_buf.data(), input_without_whitespace.data(),
                         input_without_whitespace.size(), GetModpPolicy(policy));
   }
 
@@ -109,11 +109,11 @@ bool Base64Decode(std::string_view input,
 }
 
 std::optional<std::vector<uint8_t>> Base64Decode(std::string_view input) {
-  std::vector<uint8_t> write_buf(modp_b64_decode_len(input.size()));
+  std::vector<uint8_t> write_buf(lynx_modp_b64_decode_len(input.size()));
   span<char> write = base::as_writable_chars(base::span(write_buf));
 
   size_t written_size =
-      modp_b64_decode(write.data(), input.data(), input.size());
+      lynx_modp_b64_decode(write.data(), input.data(), input.size());
   if (written_size == MODP_B64_ERROR) {
     return std::nullopt;
   }
-- 
2.49.0

