From 29fa462829741879e4ac35dea9d85b936f6f8f23 Mon Sep 17 00:00:00 2001
From: lynxtron <lynxtron@lynxjs.com>
Date: Sun, 16 Aug 2026 15:21:47 +0800
Subject: [PATCH] Revert async font manager on Windows

---
 clay/shell/common/shell.cc | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/clay/shell/common/shell.cc b/clay/shell/common/shell.cc
index 3713d58f..d8813142 100644
--- a/clay/shell/common/shell.cc
+++ b/clay/shell/common/shell.cc
@@ -560,18 +560,12 @@ bool Shell::Setup(std::unique_ptr<PlatformView> platform_view,
 
   // Setup the time-consuming default font manager right after engine created.
   if (!settings_.prefetched_default_font_manager) {
-#if defined(OS_WIN)
-    DefaultFontManagerInitializer::Instance().Request(
-        weak_engine_, task_runners_.GetIOTaskRunner(),
-        task_runners_.GetUITaskRunner(), settings_.font_initialization_data);
-#else
     fml::TaskRunner::RunNowOrPostTask(task_runners_.GetUITaskRunner(),
                                       [engine = weak_engine_] {
                                         if (engine) {
                                           engine->SetupDefaultFontManager();
                                         }
                                       });
-#endif  // defined(OS_WIN)
   }
 
   // Create FrameTimingCollector and set to engine
-- 
2.51.1.windows.1

