From 79b9edc85faff346fad1f3b304d43210ee649cc6 Mon Sep 17 00:00:00 2001
From: tangyongjie <80395553+yongjieTang@users.noreply.github.com>
Date: Thu, 13 Nov 2025 17:01:37 +0800
Subject: [PATCH] disable protobuf_full visibility

---
 BUILD.gn | 96 ++++++++++++++++++++++++++++----------------------------
 1 file changed, 48 insertions(+), 48 deletions(-)

diff --git a/BUILD.gn b/BUILD.gn
index ec3695f9..3fe5073c 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -101,69 +101,69 @@ target(_protobuf_lite_target_type, "protobuf_lite") {
 static_library("protobuf_full") {
   if (defined(build_with_chromium) && build_with_chromium) {
     # Prevent people from depending on this outside our file.
-    visibility = [
-      ":*",
+    # visibility = [
+    #   ":*",
 
-      # Used for testing protobuf generation.
-      "//base/test:proto_test_support",
+    #   # Used for testing protobuf generation.
+    #   "//base/test:proto_test_support",
 
-      # requires descriptors & reflection; testonly.
-      "//third_party/libprotobuf-mutator:*",
+    #   # requires descriptors & reflection; testonly.
+    #   "//third_party/libprotobuf-mutator:*",
 
-      # Chromecast requires descriptors and reflection.
-      "//chromecast/*",
+    #   # Chromecast requires descriptors and reflection.
+    #   "//chromecast/*",
 
-      # libassistant requires descriptors and reflection for testing.
-      "//libassistant/*",
+    #   # libassistant requires descriptors and reflection for testing.
+    #   "//libassistant/*",
 
-      # Perfetto uses the full library for testing.
-      "//third_party/perfetto/gn:protobuf_full",
+    #   # Perfetto uses the full library for testing.
+    #   "//third_party/perfetto/gn:protobuf_full",
 
-      # Some tests inside ChromeOS need reflection to parse golden files.
-      # Not included in production code.
-      "//chrome/browser/ash/child_accounts/time_limit_consistency_test:usage_time_limit_unittests",
+    #   # Some tests inside ChromeOS need reflection to parse golden files.
+    #   # Not included in production code.
+    #   "//chrome/browser/ash/child_accounts/time_limit_consistency_test:usage_time_limit_unittests",
 
-      # The protobuf-based SQLite and GPU fuzzers need protobuf_full and are not
-      # included in Chrome.
-      "//gpu:gl_lpm_fuzzer_proto",
-      "//gpu:gl_lpm_fuzzer_proto_gen",
-      "//gpu:gl_lpm_shader_to_string_unittest",
-      "//third_party/sqlite:sqlite3_lpm_corpus_gen",
+    #   # The protobuf-based SQLite and GPU fuzzers need protobuf_full and are not
+    #   # included in Chrome.
+    #   "//gpu:gl_lpm_fuzzer_proto",
+    #   "//gpu:gl_lpm_fuzzer_proto_gen",
+    #   "//gpu:gl_lpm_shader_to_string_unittest",
+    #   "//third_party/sqlite:sqlite3_lpm_corpus_gen",
 
-      # The protobuf-based Mojo LPM fuzzer needs protobuf_full and is not included
-      # in Chrome.
-      "//mojo/public/tools/fuzzers:mojolpm",
+    #   # The protobuf-based Mojo LPM fuzzer needs protobuf_full and is not included
+    #   # in Chrome.
+    #   "//mojo/public/tools/fuzzers:mojolpm",
 
-      # The root store tool is not part of Chrome itself, and needs to parse
-      # human-readable protobufs. Protobuf is stored in //net/cert however as
-      # browser needs to be able to parse serialized protobuf (which is exposed
-      # as a separate lite BUILD rule).
-      "//net/cert:root_store_proto_full",
+    #   # The root store tool is not part of Chrome itself, and needs to parse
+    #   # human-readable protobufs. Protobuf is stored in //net/cert however as
+    #   # browser needs to be able to parse serialized protobuf (which is exposed
+    #   # as a separate lite BUILD rule).
+    #   "//net/cert:root_store_proto_full",
 
-      # The spirv-fuzz fuzzer tool needs protobuf_full and is not included in
-      # Chrome.
-      "//third_party/spirv-tools/src:spirv-fuzz",
-      "//third_party/spirv-tools/src:spvtools_fuzz",
-      "//third_party/spirv-tools/src:spvtools_fuzz_proto",
+    #   # The spirv-fuzz fuzzer tool needs protobuf_full and is not included in
+    #   # Chrome.
+    #   "//third_party/spirv-tools/src:spirv-fuzz",
+    #   "//third_party/spirv-tools/src:spvtools_fuzz",
+    #   "//third_party/spirv-tools/src:spvtools_fuzz_proto",
 
-      # Some fuzzers for tint need protobuf_full and are not included in Chrome.
-      "//third_party/dawn/src/tint/*",
+    #   # Some fuzzers for tint need protobuf_full and are not included in Chrome.
+    #   "//third_party/dawn/src/tint/*",
 
-      # Dawn LPM Fuzzers
-      "//third_party/dawn/src/dawn/fuzzers:dawn_lpm_proto",
-      "//third_party/dawn/src/dawn/fuzzers:dawn_lpm_proto_gen",
+    #   # Dawn LPM Fuzzers
+    #   "//third_party/dawn/src/dawn/fuzzers:dawn_lpm_proto",
+    #   "//third_party/dawn/src/dawn/fuzzers:dawn_lpm_proto_gen",
 
-      # The Cast Core gRPC generator tool.
-      "//third_party/cast_core/public/src/build/chromium:cast_core_grpc_generator",
+    #   # The Cast Core gRPC generator tool.
+    #   "//third_party/cast_core/public/src/build/chromium:cast_core_grpc_generator",
 
-      # The proto_extras plugin generates code that some of the above fuzzers
-      # use, and thus needs to know the full types.
-      "//components/proto_extras:protobuf_full_support",
+    #   # The proto_extras plugin generates code that some of the above fuzzers
+    #   # use, and thus needs to know the full types.
+    #   "//components/proto_extras:protobuf_full_support",
 
-      # Sync requires descriptors for testing.
-      "//components/sync:sync_full_proto_unittests",
-      "//components/sync/protocol:full_proto",
-    ]
+    #   # Sync requires descriptors for testing.
+    #   "//components/sync:sync_full_proto_unittests",
+    #   "//components/sync/protocol:full_proto",
+    # ]
   }
 
   deps = [ ":utf8_range" ]
-- 
2.50.1 (Apple Git-155)

